metadata_tools/hosts/GO_0xxx/GO_0xxx_index.py

```
@@ -1,46 +1,37 @@
-#
-#  2) Copy and rename the supplemental label template from an existing host,
-#     e.g.:
-#
-#         hosts/GO_0xxx/templates/GO_0xxx_index_supplemental.lbl
-#
-#  3) Follow the instructions in the supplemental label template file.
-#
-################################################################################
-import host_init
+##########################################################################################
+import host_init    # noqa: F401 (imported for side effects)
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)